lib/liquid/context.rb

```diff
@@ -27,7 +27,7 @@ def initialize(environments = {}, outer_scope = {}, registers = {}, rethrow_erro
       @this_stack_used = false
 
       if rethrow_errors
-        self.exception_handler = ->(e) { raise }
+        self.exception_handler = ->(e) { raise e }
```

Comment:
I think it's better to not interrupt the filter chain. I allows us to get the most successful render result, i.e. all defined filters will be applied and undefined filters will be just skipped. At least this is what we have now, without the changes in this PR.
But it's up to you to decide, which approach is better here.